test/functional/feature_graphene_active.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python3
+# Copyright (c) 2019 The Unit-e developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/licenses/MIT.
 # Copyright(c) 2018 The Unit - e developers
```

Comment:
should this not be `2018-2019`?

Reply:
This file has a malformed header which is not detected by the script. I'll fix this and check why the script didn't detect the malformed header.

Reply:
The second notice was lacking a space between `Copyright` and `(c)`, that's why the script didn't pick it up. Fixed now.

Reply:
Maybe we could improve the regular expression that is used in this line:

https://github.com/dtr-org/unit-e/pull/1069/files#diff-dc87c4c96fc5f98c3734a2c16504eb30R354

writing something like:
```python
COPYRIGHT = 'Copyright\s?\([cC]\)'
```
or
```python
COPYRIGHT = 'Copyright\s*\([cC]\)'
```
instead of 
```python
COPYRIGHT = 'Copyright \([cC]\)'
```

Reply:
I'm also wondering if we should, just in case, add support for the ® symbol, given that UNICODE is being used more & more as time passes.

Reply:
Improving the regular expression is a good idea. I will do that.

Support for the unicode copyright symbol I would add when we or upstream start using it. It's not a big deal if a header is not recognized because of that. It shows up as a file without copyright instead of reporting the proper copyright line. These files need special attention anyway.